docs/diagrams/AddWeddingSequenceDiagram.puml

```diff
@@ -0,0 +1,46 @@
+@startuml
```

Comment:
![Screenshot 2024-10-27 at 11 47 19 AM](https://github.com/user-attachments/assets/017d8daf-f69e-43b0-85d0-730726af3946)
this part looks a bit confusing, i think it would be clearer if its like this
![IMG_4490](https://github.com/user-attachments/assets/01bf6f39-10e8-4853-8be3-20785255cea1)

Reply:
PlantUML cuts the activation bars, shouldnt the result be returned within the alt?